### DISCUSSION TITLE
Compile issue in quickstart: reportWebVitals.ts file

### DISCUSSION THREAD

**Discussion by dredlong** [twilio-labs/paste#1193] [Q&A] [answered]
Hi - I was just trying to follow the quickstart setup to play around with Paste for a React app project. Following the quickstart instructions (using Yarn v1.22.10) I get a compile error:

```
src/reportWebVitals.ts
  Line 5:5:  Definition for rule 'promise/catch-or-return' was not found  promise/catch-or-return
  Line 7:7:  Definition for rule 'promise/always-return' was not found    promise/always-return
```

It seems like it's related to ESLint? Is there an extra step needed in the install?

**Comment by SiTaggart** [comment, marked as answer]
Hi!

No, there's no extra steps, just a small mistake on our behalf.

In that report web vitals file, you should see two commented out lines, 5 and 7 by the looks of it, which try to disabled a lint rule. 

The rules they relate to don't exist outside of our original repo. 

Just delete the commented out lines and you should be away.

Let us know what you're building!

**Reply by dredlong** [reply]
Thanks - thought I should try Paste out for my App Bootcamp demo. May drop in the channel when it's closer to being finished.
